cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -42,79 +52,102 @@
      */
     private static final String DOCKER_PASSWORD_VAR_NAME = "CF_DOCKER_PASSWORD";
 
+    /**
+     * Name of the key of label for metadata.
+     */
+    private static final String METADATA_KEY = "CF_METADATA_KEY";
```

Comment:
This constant is also defined in the ApplicationBean class. It would be better to have it only at a single place